.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
```

Comment:
that's what I am going in tchannel-python, as it already uses the matrix. Not quite sure how to do it in Go.